aleph/views/base_api.py

```diff
@@ -51,6 +51,7 @@ def _metadata_locale(locale):
             "version": __version__,
             "banner": settings.APP_BANNER,
             "ui_uri": settings.APP_UI_URL,
+            "messages_url": settings.APP_MESSAGES_URL,
```

Comment:
What do we prefer? URI or URL?

Reply:
URL is fine, it seems consistent with the settings portion of the line above, which isn't actually consistent with itself, hmmm.